<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-2GC0944" failures="1" tests="4" name="Test" time="0.114" errors="0" timestamp="2025-07-02T23:02:46 IST">
  <testcase classname="com.test.SampleTest" name="testFail" time="0.023">
    <skipped/>
  </testcase> <!-- testFail -->
  <testcase classname="com.test.SampleTest" name="testFail" time="0.002">
    <skipped/>
  </testcase> <!-- testFail -->
  <testcase classname="com.test.SampleTest" name="testFail" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.test.SampleTest.testFail(SampleTest.java:18)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- testFail -->
  <testcase classname="com.test.SampleTest" name="testPass" time="0.002"/>
</testsuite> <!-- Test -->
